samples/VestingTransactions.examples.ocf.json

```diff
@@ -0,0 +1,19 @@
+{
+  "file_type": "OCF_TRANSACTIONS_FILE",
+  "items": [
+    {
+      "object_type": "TX_VESTING_EVENT",
+      "id": "d0a02b7a",
+      "security_id": "vesting-ex-1",
+      "date": "2022-07-14",
+      "vesting_condition_id": "qualifying-sale"
+    },
+    {
+      "object_type": "TX_VESTING_START",
+      "id": "08f8b870",
+      "security_id": "vesting-ex-1",
+      "date": "2021-01-01",
+      "vesting_condition_id": "vesting-start"
+    }
+  ]
+}
```

Comment:
These example files are referenced from the documentation, but are included here in samples so that the validator checks them.